lib/instrumental/agent.rb

```diff
@@ -81,6 +82,9 @@ def initialize(api_key, options = {})
       @queue = Queue.new
 
       setup_cleanup_at_exit if @enabled
+
+      Metrician.activate(self)
```

Comment:
I've added and option and some tests around it. I'm ready to cut a new release and move to next steps assuming you're on board.